src/lib/Theme.ts

```diff
@@ -377,7 +377,7 @@ export default class Theme {
                 domainURL =`https://${currentContext.env}`
             const url = new URL(domainURL);
             const hostName = url.hostname;
-            let domain = hostName.replace('api.', '');
+            let domain = hostName.replace('api', 'platform');
```

Comment:
Add comment here why we are doing this with example

Reply:
Okay